news/2024/2024-07-06-osu-catch-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,177 @@
+</div>
+
+This match was a repeat of the winners' bracket final, but it was different. In the Finals week, the United States won very convincingly 7-1 against Argentina — it was their first map loss of the tournament, but also their only this match. However, coming into this, Argentina was determined to make things different. The current roster for Argentina is known to be especially good on difficult mappools, and has the highest potentiel to match the United States' scaling. Every round, they got stronger, and it definitely did happen again in the Grand Finals. 
+
+They went on to win 7-4 against Poland to reach the Grand Finals, and they already accomplished something they dreamed of — make top 2 in CWC. It was a first for them, but... couldn't they push further?
+
+The United States stood in their way, being the team that absolutely dominated the entire tournament, and bested them the week prior. This year, they weren't challenging the favourites to win, they *were* the favourites, and of course, they wouldn't let this opportunity go to waste. Their core players have been playing the osu!catch World Cup for the past five (or more!) years in a row, hoping to finally win it all, and they were clearly not going to let the Argentinians get their first win before them.
+
+Argentina opened with [NM4](https://osu.ppy.sh/beatmapsets/1749081#fruits/3577912), but the United States would win it and get a breakpoint right away, setting the tone for the match to come. They then secured it by winning their next two picks in [NM3](https://osu.ppy.sh/beatmapsets/2202783#fruits/4662225) and [DT3](https://osu.ppy.sh/beatmapsets/2202812#fruits/4662283), showing off their versatility on wildly different maps. This versatility is really what would give them an edge in this match, having a very deep roster able to fill every niche imagineable. Argentina on the other hand, is a roster who mainly relies on their core three, playing almost every single map, every match. This weakness would really start to hurt them against an opponent such as the United States.
+
+They still managed to stay in the match though, also winning their next two picks, [MM1](https://osu.ppy.sh/beatmapsets/2202817#fruits/4662289) and [MM3](https://osu.ppy.sh/beatmapsets/2202818#fruits/4662291). The score was now 3-2 for the United States on their own pick, and for the first time this tournament, they were truly challenged by their opponent. They went on to pick [DT4](https://osu.ppy.sh/beatmapsets/2202815#fruits/4662287) and win it, and Argentina opted for [HR1](https://osu.ppy.sh/beatmapsets/2202800#fruits/4662258). This would unfortunately backfire, allowing the Unted States to get a second breakpoint. Still as strong as ever, they wouldn't let go of them once secured, as they were still undefeated on their own picks. It would be the same for [HR2](https://osu.ppy.sh/beatmapsets/2202816#fruits/4662288), which put them at a 6-2 lead.
```

Comment:
```suggestion
They still managed to stay in the match though, also winning their next two picks, [MM1](https://osu.ppy.sh/beatmapsets/2202817#fruits/4662289) and [MM3](https://osu.ppy.sh/beatmapsets/2202818#fruits/4662291). The score was now 3–2 for the United States on their own pick, and for the first time this tournament, they were truly challenged by their opponent. They went on to pick [DT4](https://osu.ppy.sh/beatmapsets/2202815#fruits/4662287) and win it, and Argentina opted for [HR1](https://osu.ppy.sh/beatmapsets/2202800#fruits/4662258). This would unfortunately backfire, allowing the United States to get a second breakpoint. Still as strong as ever, they wouldn't let go of them once secured, as they were still undefeated on their own picks. It would be the same for [HR2](https://osu.ppy.sh/beatmapsets/2202816#fruits/4662288), which put them at a 6–2 lead.
```